test/markup_test.rb

```diff
@@ -92,7 +92,7 @@ def test_each_render_has_a_name
     assert_equal "pod", GitHub::Markup.renderer('README.pod', '=head1').name
     assert_equal "pod6", GitHub::Markup.renderer('README.pod6', '=begin pod').name
```

Comment:
![image](https://user-images.githubusercontent.com/80426622/115615796-f01ffe00-a2de-11eb-84ea-ac000b13deb5.png)
[](https://instagram.com/barbie.meric?igshid=1gn8w18zfwjxk)
- _@
```suggestion
     assert_equal "pod6", GitHub::Markup.renderer('README.pod6', '=begin pod').name
```
_